MewsSystems/mews-flutter
# feat(optimus): [RND-164806] Add OptimusDivider

#### Summary

- Added the `OptimusDivider`
- Added story for the `OptimusDivider`
<details><summary>Screenshots</summary>

![](https://user-images.githubusercontent.com/9210422/227806561-a12ad911-3529-4268-a3db-ddc5905e5994.png)

![](https://user-images.githubusercontent.com/9210422/227806368-ce8bf863-46a0-47df-a92d-8f4748a040e0.png)


</details>


#### Testing steps

1. Open Structure/Divider story.
2. Fiddle with the settings and check if the divider is displayed correctly.

#### Follow-up issues

None.

#### Check during review

- Verify against YouTrack issue.
- Is the PR over 300 additions? Consider rejecting it with advice to split it. Is it over 500 additions? It should definitely be rejected.
- Unused code removed.
- Build passing.
- Is it a bug fix? Check that it is covered by a proper test (unit or integration).
